src/CTA.Rules.Update/RulesCli.cs

```diff
@@ -13,8 +13,8 @@ public class Options
         [Option('s', "solution-path", Required = false, HelpText = "Solution file path.")]
         public string SolutionPath { get; set; }
 
-        [Option('r', "rules-input-infile", Required = false, HelpText = "Rules json input file")]
-        public string RulesInputFile { get; set; }
+        [Option('r', "rules-input-infile-directory", Required = false, HelpText = "Directory containing rules json input file(s)")]
+        public string RulesInputFilesDirectory { get; set; }
```

Comment:
nit: Same as comment for PortCore project

Reply:
Will do